src/richie/apps/courses/factories.py

```diff
@@ -375,6 +375,29 @@ def with_courses(self, create, extracted, **kwargs):
             self.courses.set(extracted)
 
 
+class LicenceLogoImageFactory(factory.django.DjangoModelFactory):
```

Comment:
I propose that you inherit from FilerImageFactory and just override the file method...

Reply:
@sampaccoud i've got a hardtime to do it but it's ok now